perf_test/BenchmarkMain.cpp

```diff
@@ -21,6 +21,12 @@
 #include <Benchmark_Context.hpp>
 #include <Kokkos_Core.hpp>
 
+static void BM_StringCreation(benchmark::State& state) {
+  for (auto _ : state) std::string empty_string;
+}
+
+BENCHMARK(BM_StringCreation)->ArgNames({"N", "R"})->Args({100'000, 1'000});
```

Comment:
Note: this can be dropped once #1636 is merged. Without any benchmark registered, `benchmark::Initialize` complains about no benchmarks detected.

Reply:
Removed.